google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerMetadataProvider.java

```diff
@@ -28,7 +28,7 @@
 class SpannerMetadataProvider {
   private final Map<Metadata.Key<String>, String> headers;
   private final String resourceHeaderKey;
-
+  private static final String routeToLeaderHeaderKey = "x-goog-spanner-route-to-leader";
```

Comment:
nit: Java `static final` fields are considered constants, and should be named using `UPPER_SNAKE_CASE`.

Reply:
Done.